src/main/java/frc/robot/hardware/encoder/EmptyEncoder.java

```diff
@@ -0,0 +1,10 @@
+package frc.robot.hardware.encoder;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+
+public class EmptyEncoder implements IEncoder {
```

Comment:
wdym EmptyEncoder?

Reply:
(its dror)if u like have a swerve sim or somth that gets an encoder but you dont have so ull use this instead (i dont fully understand this either)